### ISSUE TITLE
[FEATURE] Binary file download

### ISSUE THREAD

**Issue by majkinetor** [usebruno/bruno#539]
It should be possible to save a binary file locally when `content-disposition` header is present

<details><summary>details</summary> 

![image](https://github.com/usebruno/bruno/assets/85767/bb0a5784-2a0c-49a5-9299-9cf95b8f6c60)

![image](https://github.com/usebruno/bruno/assets/85767/3ef9af7b-a844-4bfd-a564-d7728b54fce2)

</details>

**Comment by martinsefcik** [comment]
I think it should be possible to store response body to file for any type of response and even if `content-disposition` header is not present (it can be just hint for filename during saving if such header present).

**Comment by majkinetor** [comment]
Indeed. However, its totally needed for binary files because you just cant see response at all. 

I generated shell code for my case to be able to see it until we eventually have this:

```
script:post-response {
  var path = require("path");
  const HOST = bru.getEnvVar("HOST");
  bearer = req.getHeader("Authorization");
  let cd = res.getHeader('content-disposition');

  var match = cd.match(/(filename=")(.*)";.+$/);
  var filename = match && match[2] || 'default-filename.out';
  var filename = path.basename(filename)
  let url = req.getUrl().replace("{{HOST}}", HOST)
  let method = req.getMethod();
  console.log(`rm "${filename}"; curl.exe ${url} --request ${method} --header "Authorization:  ${bearer}" -O -J; start ${filename}`);
}
```

I use CTRL + I to copy/paste it into shell

**Comment by martinsefcik** [comment]
@helloanoop I guess this issue was implemented as part of PR #766 and can be closed.

**Comment by helloanoop** [comment]
Closing this issue. @majkinetor please reopen if you encounter any issues with the functionality.

**Comment by majkinetor** [comment]
The only issue I found is that larger PDF files make bruno unusable (e.g. 500 pages). It takes minutes to render during which time bruno is frozen. The same PDF opens instantly in browser.

**Comment by martinsefcik** [comment]
> The only issue I found is that larger PDF files make bruno unusable (e.g. 500 pages). It takes minutes to render during which time bruno is frozen. The same PDF opens instantly in browser.

But I guess it is caused by pdf file preview feature implemented in PR #982 implemented by @n00o (and not by binary file download feature implemented in PR #766) as it consumes much more resources in case of big pdf files and renders big HTML content.

Probably it should be limited during rendering to render only few pages or it should render only one page at once and there should be some prev/next page buttons or pages should be dynamically loaded when you scroll down pdf content. But for this another issue should be created as it i not related to this issue.

**Comment by majkinetor** [comment]
OK @martinsefcik , I will open bug issue.
